services/release-hatch/justfile

```diff
@@ -0,0 +1,74 @@
+export SSL_CERT := "certs/release-hatch.crt"
+export SSL_KEY := "certs/release-hatch.key"
+export SELFSIGNED_CERT := "certs/testing.crt"
+export SELFSIGNED_KEY := "certs/testing.key"
+export SYSTEM_CERTS := "/usr/local/share/ca-certificates/release-hatch"
+
+[private]
+default:
+  @just --list
+
+# pull latest version and deploy
+deploy: && restart
+    echo "Pulling image"
+    docker-compose --no-ansi pull --quiet release-hatch
+
+# restart release-hatch
+restart *args:
+    docker-compose --no-ansi up --no-build {{ args }} --detach release-hatch
+
+# update to the certificate for release-hatch from backends.opensafely.org
+update-certs:
```

Comment:
When/why would this get called?

Reply:
Good point! It should be called as part of the systemd timer, but I omitted that. Will fix.

Reply:
Hmm, although, doing this is tricky, as it will break the self signed teset set up

Reply:
This is fixed now, and 90% covered by tests